spec/bat/stemcell_spec.rb

```diff
@@ -88,6 +88,9 @@
       'bosh-warden-boshlite-ubuntu-trusty-go_agent' => false,
       'bosh-warden-boshlite-centos-go_agent' => false,
 
+      # Azure CPI does not support network reconfig
```

Comment:
azure cpi does support network reconfig: https://github.com/AbelHu/bosh-azure-cpi-release/blob/develop/src/bosh_azure_cpi/lib/cloud/azure/cloud.rb#L188. it's just not efficient but the tests should pass anyway.